src/main/java/com/mycompany/exercisethree/BasicCalculator.java

```diff
@@ -0,0 +1,57 @@
+package com.mycompany.exercisethree;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+/**
+ *
+ * @author usuario
+ */
+public class BasicCalculator extends Calculator {
+
+    @Override
+    public void operateCalculator() {
+
+        Scanner getInt = new Scanner(System.in);
+        System.out.println("WELCOME TO THE BASIC CALCULATOR, PLEASE CHOOSE WHAT YOU WANT TO DO ");
+        System.out.println("1. ADD");
+        System.out.println("2. SUBSTRACT");
+        System.out.println("3. MULTIPLY");
+        System.out.println("4. DIVIDE");
+        System.out.println("5. RETURN");
+
+        try {
+            option = getInt.nextInt();
+            listNumbers = (option >= 1 && option <= 4) ? getInputString() : null;
+        } catch (InputMismatchException e) {
+            System.out.println("Invalid option, please enter a valid option");
+            getInt.nextLine(); // clear the scanner buffer
+            operateCalculator();
+        }
+
+        switch (option) {
+            case 1 -> {
+                System.out.println("The result is: " + sumList(listNumbers) + '\n');
```

Comment:
Question if you are using a global variable listNumbers and the parent has it, why are you passing this?

So my suggestion is make listNumbers as local variable